tests/jest/setupTests.js

```diff
@@ -6,4 +6,4 @@ import pkg from '../../package.json';
 Enzyme.configure({ adapter: new Adapter() });
 
 // global test variables
-global.SDK_VERSION = `ik-sdk-version=react-${pkg.version}`;
+global.SDK_VERSION = `ik-sdk-version=javascript-1.5.2`;
```

Comment:
Please remove SDK version from all places. You can pass an empty string while initialising JS SDK to ensure it is not added in the URL.
```
const ik = new ImageKit({...initializationParams, sdkVersion: ""})
```

Reply:
ok. let's do this.